spring-cloud-app-broker-acceptance-tests/src/test/java/org/springframework/cloud/appbroker/acceptance/CreateInstanceWithServicesAcceptanceTest.java

```diff
@@ -90,11 +90,16 @@ void deployAppsAndCreateServicesOnCreateService() {
 		// when the service instance is deleted
 		deleteServiceInstance(SI_NAME);
 
-		// service has no applications bound to it
-		ServiceInstance serviceInstance2AfterDeletion = getServiceInstance(BACKING_SI_2_NAME);
-		assertThat(serviceInstance2AfterDeletion.getApplications()).isEmpty();
-
-		deleteServiceInstance(BACKING_SI_2_NAME);
+		// and the backing services are deleted
+		assertThat(listServiceInstances()).doesNotContain(BACKING_SI_1_NAME);
+		assertThat(listServiceInstances()).doesNotContain(BACKING_SI_2_NAME);
+
+		// TODO: another story to only remove the instances with service definition specified
```

Comment:
Could you add a link to the story as part of the TODO?

Reply:
Good idea! I will create a story and update the TODO.